test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -1214,8 +1261,10 @@ private String firstTimeStampFor(int initialTZOffset, int requestTZOffset, Sched
         Map<String,DateTime> events = Maps.newHashMap();
         events.put("enrollment", ENROLLMENT);
         
-        return new ScheduleContext.Builder().withStudyIdentifier(TEST_STUDY).withInitialTimeZone(DateTimeZone.UTC)
-                .withStartsOn(NOW).withAccountCreatedOn(ENROLLMENT.minusHours(2)).withEndsOn(endsOn)
+        DateTimeZone zone = DateTimeZone.forOffsetHours(-7); // zone of the original times
+        // Initial time zone and the time zone of enrollment should be the same. They are in reality.
+        return new ScheduleContext.Builder().withStudyIdentifier(TEST_STUDY).withInitialTimeZone(zone)
+                .withStartsOn(NOW).withAccountCreatedOn(ENROLLMENT.minusHours(2).withZone(zone)).withEndsOn(endsOn)
```

Comment:
I'm not sure if we should be changing the timezone here. iirc, we wrote the tests like this specifically to ensure that changing timezones doesn't break anything. This is being used by a lot of tests, and I'm not sure what the impact is of changing this.